### ISSUE TITLE
Update documentation for dynamic tabbed interfaces that use `.nav-item` to set role on `li`

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Framework#220]
**Describe the bug**

https://ucf.github.io/Athena-Framework/components/navs/#javascript-behavior-for-dynamic-tabbed-interfaces

Examples within the section linked above that utilize `.nav-item`s (use `<ul>`/`<li>` instead of `<div>`/`<a>`) should be updated to set `role="presentation"` on each `<li>`.  The implicit `role="listitem"` currently conflicts with `role="tablist"` on the parent `.nav`s.

**Steps to Reproduce**

n/a

**Expected Behavior**

Our markup examples should not get flagged in accessibility scans for not meeting required-owned elements rules (e.g. "container element is empty").

**Possible Fix**

Update instances within the documentation where this convention is used.

**Your environment:**

n/a

**Additional context**

n/a
